### PR TITLE
Add env var for binaries_version

### PR DESCRIPTION
**Pull request by diserere**
Prooflink to integration tests with environment variable used (implemented in branch),
TON_SDK_BIN_VERSION set to '0_23_99',
actual binary version is '0.23.102':
https://builder.tonlabs.io/jenkins/job/Integration/job/integration-tests/job/fix-binaries-version/5/execution/node/95/log/
see in log:
```   console.log
     Client version 0.23.0 uses compatible binaries version: 0.23.99 
     (requested version: 0.23.1; calculated version: 0_23_99)
```

### CODE REVIEW

File: __tests__/_/binaries.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ const zlib = require('zlib');
 
 
 export const p = os.platform();
-export const bv = (binaries_version || version).split('.')[0];
+export const bv = process.env.TON_SDK_BIN_VERSION ? process.env.TON_SDK_BIN_VERSION : (binaries_version || version).split('.')[0];
```
Comment (melsomino) [inline __tests__/_/binaries.js:11]:
```suggestion
export const bv = process.env.TON_SDK_BIN_VERSION || binaries_version || version.split('.')[0];
```

Comment (diserere) [reply __tests__/_/binaries.js:11]: Should it be 
`process.env.TON_SDK_BIN_VERSION || (binaries_version || version).split('.')[0]` ?
Cause `binaries_version` also should be truncated to major number, as far as i understand

Comment (diserere) [reply __tests__/_/binaries.js]: Fixed in last commit, +removed `binaries_version` from package as obsolete

**Comment by diserere** [comment]
Tests after changes requested:
- with `TON_SDK_BIN_VERSION` not set:
```   console.log
     Client version 0.23.1 uses compatible binaries version: 0.23.102 
     (requested version to download: 0)
```
https://builder.tonlabs.io/jenkins/job/Integration/job/integration-tests/job/master/2356/execution/node/95/log/
- with `TON_SDK_BIN_VERSION` set to `0_23_99`:
```   console.log
     Client version 0.23.0 uses compatible binaries version: 0.23.99 
     (requested version to download: 0_23_99)
```
https://builder.tonlabs.io/jenkins/job/Integration/job/integration-tests/job/fix-binaries-version/6/execution/node/95/log/

**Pull request merged**